micro-rdk/src/common/camera.rs

```diff
@@ -1,12 +1,23 @@
-#![allow(dead_code)]
-use std::sync::{Arc, Mutex};
-
-use crate::proto::component::camera;
-use bytes::{Bytes, BytesMut};
+use super::{
+    generic::DoCommand,
+    status::{Status, StatusError},
+};
+use crate::{google, proto::component::camera};
+use bytes::BytesMut;
 use prost::Message;
-
+use std::{collections::HashMap, sync::Mutex};
 use thiserror::Error;
 
+#[cfg(feature = "camera")]
+use {
+    super::{config::ConfigType, registry::ComponentRegistry, registry::Dependency},
+    std::sync::Arc,
+};
+
+pub static COMPONENT_NAME: &str = "camera";
+#[cfg(feature = "camera")]
+pub static FAKE_JPEG: &[u8] = include_bytes!("../../../etc/assets/symbol.jpg");
```

Comment:
Any reason not to just put this in the same directory as `camera.rs`, or under an `assets` directory there rather than at the top?

And `symbol` is a pretty generic name. Maybe `fake_camera_image`?.

Reply:
I was avoiding putting artifacts directly in the micro-rdk core repo.
fine with putting it here though. 

I'm tempted to move the FakeCamera and the image into the native directory and leave just the trait definition here. 

As for the name, I had several candidate files when testing, (symbol, checkers, etc). will rename it

Reply:
Yes, it should live next to whatever ends up using it, wherever that lives.